backends/utils.py

```diff
@@ -78,3 +79,18 @@ def has_social_auth(user, provider):
         log.info('No social auth for %s for user %s', provider, user.username)
         return False
     return True
+
+
+def get_staff_access_payload():
```

Comment:
More accurately these are credentials:
```suggestion
def get_staff_edx_client_credentials():
```